server/devdocs/bin/generate-proptypes-index.js

```diff
@@ -106,7 +199,8 @@ const writeFile = ( contents ) => {
 	fs.writeFileSync( path.join( root, 'server/devdocs/proptypes-index.json' ), JSON.stringify( contents ) );
 };
 
-const main = ( () => {
+( () => {
```

Comment:
Technically, this <abbr title="immediately invoked function expression">IIFE</abbr> doesn't add anything except for some visual separation. This is assumedly my personal preference, but I'd keep the named `main` function, and would perhaps move it to the top (or near the top) of the file, then leave its invocation at the bottom. Having `main` at the top gives readers an immediate outline of what the program will perform.